ugvc/pipelines/mrd/srsnv_training.py

```diff
@@ -183,6 +213,21 @@ def parse_args(argv: list[str]) -> argparse.Namespace:
         default=None,
         help="""random seed for reproducibility""",
     )
+    parser.add_argument(
+        "--load_dataset_and_model",
+        action="store_true",
+        help="""If flag is provided, skip dataset genenration and model training.
+        Output folder should contain featuremap_df.parquet and model.joblib,
+        and all other relevant files.""",
+    )
```

Comment:
Wouldn't it be better to give the model and dataset paths as input rather than require that they already be saved in the right path?